server/hedley/modules/custom/hedley_admin/hedley_admin.module

```diff
@@ -431,143 +431,86 @@ function hedley_admin_administrators_access() {
   }
 
-  $query = new EntityFieldQuery();
-  $query->entityCondition('entity_type', 'node')
+  $base_query = (new EntityFieldQuery())->entityCondition('entity_type', 'node')
     ->entityCondition('bundle', 'resilience_survey')
     ->propertyCondition('status', 1);
 
-  $result = $query->execute();
-
-  if (empty($result['node'])) {
-    drupal_set_message(t('No survey results available.'), 'warning');
-    return [];
-  }
-
-  $survey_nids = array_keys($result['node']);
-  $survey_nodes = node_load_multiple($survey_nids);
+  $count = clone $base_query;
+  $count = $count->count()->execute();
 
   $rows = [];
-  $surveys = [];
-
-  foreach ($survey_nodes as $node) {
-    $nurse_field = field_get_items('node', $node, 'field_nurse');
-    $nurse_id = !empty($nurse_field) ? $nurse_field[0]['target_id'] : NULL;
-
-    $health_center_name = 'Unknown Center';
-    if ($nurse_id) {
-      $nurse_node = node_load($nurse_id);
-      if ($nurse_node) {
-        $health_center_field = field_get_items('node', $nurse_node, 'field_health_centers');
-        if (!empty($health_center_field)) {
-          $health_center_id = $health_center_field[0]['target_id'];
-          $health_center_node = node_load($health_center_id);
-          if ($health_center_node) {
-            $health_center_name = $health_center_node->title;
-          }
-        }
-      }
-    }
-
-    $survey_type_field = field_get_items('node', $node, 'field_resilience_survey_type');
-    $survey_type = !empty($survey_type_field) ? $survey_type_field[0]['value'] : 'Unknown';
-
-    $date_measured_field = field_get_items('node', $node, 'field_date_measured');
-    $date_measured = !empty($date_measured_field) ? date('Y-m-d', strtotime($date_measured_field[0]['value'])) : 'Unknown';
 
-    $nurse_name = 'Unknown Nurse';
-    if ($nurse_id) {
-      $nurse_node = node_load($nurse_id);
-      if ($nurse_node) {
-        $nurse_name = $nurse_node->title;
-      }
-    }
-
-    $survey_answers_field = field_get_items('node', $node, 'field_resilience_survey_signs');
-    $survey_answers_raw = '';
-
-    if (!empty($survey_answers_field)) {
-      $survey_answers_raw = implode('|', array_column($survey_answers_field, 'value'));
-    }
+  while ($offset < $count) {
+    $query = clone $base_query;
+    $result = $query->range($offset, $batch_size)->execute();
 
-    $survey_answers = explode('|', $survey_answers_raw);
-    $answers_processed = [];
+    if (empty($result['node'])) return [];
 
-    foreach ($survey_answers as $answer_pair) {
-      $parts = explode('-', $answer_pair);
-      if (count($parts) === 2) {
-        list($question, $answer) = $parts;
-        $question_number = str_replace('q', '', $question);
+    foreach (node_load_multiple(array_keys($result['node'])) as $node) {
+      $nurse_id = field_get_items('node', $node, 'field_nurse')[0]['target_id'] ?? NULL;
+      $nurse_name = $nurse_id ? (node_load($nurse_id)->title ?? 'Unknown Nurse') : 'Unknown Nurse';
```

Comment:
Again, too much in one row. Separate for better readability.
Also `??` is php 8 operator. We're using php 7. Can't use it.